### PR TITLE
Task5: redux adding

### PR DESCRIPTION
**Pull request by BGRTi**

### CODE REVIEW

File: components/movie-page/movie-page.jsx [hunk 1]
```diff
@@ -1,31 +1,65 @@
...
-          description: 'Some text about film',
-          genres: ['drama', 'ne drama'],
-        },
-      ],
+      films: {},
     };
   }
 
+  componentDidMount = () => {
+    console.log(2);
```
Comment (YauheniRohau) [inline components/movie-page/movie-page.jsx]: remove

File: components/main/main.jsx [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
     this.state = {
-      someState: 'Movies',
+      movies: [],
     };
   }
 
   render() {
-    const { someState } = this.state;
+    const { movies } = this.props;
+    console.log(this.props);
```
Comment (YauheniRohau) [inline components/main/main.jsx]: remove

File: components/filters/filters.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = { value: '' };
+  }
+
+  handleSubmit = (searchBy) => {
+    const { getSearchByProp } = this.props;
+    const { value } = this.state;
+    // this.setState({ value: getSearchByProp(searchBy) });
```
Comment (YauheniRohau) [inline components/filters/filters.jsx]: remove

File: package.json [hunk 4]
```diff
@@ -17,7 +17,9 @@
     "react-redux": "^6.0.1",
     "react-router": "^4.4.0",
     "react-router-dom": "^5.0.0",
+    "react-thunk": "^1.0.0",
```
Comment (YauheniRohau) [inline package.json:21]: Why do you need it?

Comment (BGRTi) [reply package.json:21]: that was added by mistake, i needed redux-thunk

File: components/filters/filters.jsx (hunk 3 above)
Comment (BGRTi) [reply components/filters/filters.jsx]: done

File: components/main/main.jsx (hunk 2 above)
Comment (BGRTi) [reply components/main/main.jsx]: done

File: components/movie-page/movie-page.jsx (hunk 1 above)
Comment (BGRTi) [reply components/movie-page/movie-page.jsx]: done

**Pull request merged**